src/index.ts

```diff
@@ -73,6 +74,13 @@ client.on('ready', () => {
     setRandomActivity(client);
   });
   activityCronJob.start();
+
+  // setup voice channel names
+  setRandomVoiceChannelNames(client);
+  const voiceChannelNamesCronJob = new CronJob('* * * * *', () => {
```

Comment:
TODO switch to every Sunday at midnight

Reply:
I think you want "0 0 0 ? * SUN *"  (source: http://www.cronmaker.com/?1)

Reply:
`0 0 0 * SUN *` i think